MANUAL.txt

```diff
@@ -4093,7 +4101,7 @@ Hubert Plociniczak,
 Ilya V. Portnov,
 Ivo Clarysse,
 J. Lewis Muir,
-Jaime Marquínez Ferrándiz,
+Jaime Marquínez Ferrándiz,
```

Comment:
Probably in one case it's a combining character accent; in the other case a combined accented character.  The latter (where you just need one backspace to delete both) is probably better; still, this doesn't belong with the other change in this PR, so can you revert this part?